platform-version-prereq.cmake

```diff
@@ -1,22 +0,0 @@
-###
```

Comment:
with a submodule build, we control the git reference that our submodule points too.  So having a platform version prereq file is unnecessary.